website/templates/include/comment_template.mako

```diff
@@ -21,14 +31,18 @@
                     </div>
                 </div>
 
-                <div data-bind="if: isAbuse">
+                <div data-bind="if: isAbuseNotDeleted">
                     <div>
-                        <span data-bind="if: hasChildren()">
+                        <span class="text-muted">
+                            <em>Comment reported.</em>
+                        </span>
+                        <span data-bind="if: hasChildren()" class="comment-actions pull-right">
                             <i data-bind="css: toggleIcon, click: toggle"></i>
                         </span>
-                        Comment reported.
                     </div>
-                    <a data-bind="click: submitUnreportAbuse">Not abuse</a>
+                    <div data-bind="if: hasReport">
```

Comment:
No, has report refers to the person viewing the comments. If they have a report submitted then they can remove the report. @mfraezz